schema.json

```diff
@@ -83,6 +83,10 @@
                         "type": "string",
                         "minLength": 1
                     }
+                },
+                "role": {
+                    "type": "string",
+                    "enum": ["client", "server"]
```

Comment:
What about the "universal" case (ie. "*") that we discussed yesterday?

Reply:
> I think we need to rework the examples somewhat. Right now we only have fio examples and so you are adding client/server roles to those examples to demonstrate what is possible 

We should probably show examples for fio and uperf only with client role, and show trafficgen with client and server

Reply:
I wasn't sure what to call it, "all" is fine with me.  It just seemed pretty obvious that we should have something that allows us to not list parameters multiple times so that they can go to multiple roles.

Reply:
> > I think we need to rework the examples somewhat. Right now we only have fio examples and so you are adding client/server roles to those examples to demonstrate what is possible
> 
> We should probably show examples for fio and uperf only with client role, and show trafficgen with client and server

Am I being dense when I ask what does uperf with only a client role mean?

Reply:
It means there are currently no server params for uperf, in spite of it being a client-server benchmark.  If we add server options (maybe we want to add an option to NUMA pin the server), then we can add an example.  The overall point boing that just because you have a client-server benchmark does not mean there will always be server params.

Reply:
So...I was being dense :-)